test/shared/helpers/view.test.js

```diff
@@ -90,19 +94,49 @@ describe('view', function () {
     context('when extractFetchSummary returns a model', function () {
       BaseViewStub.extractFetchSummary.is(function () {
         return {
-          model: {
-            model: 'SomeModel',
+          some_model: {
+            model: 'ModelClass',
+            id: 1
+          }
+        };
+      });
+
+      it('includes the fetch summary for the model and does not include a data-attribute for the model', function () {
+        var result = subject('test', {
```

Comment:
this is pretty nit-picky, but i think we should be invoking the subject with a scope that has the `_app` attribute set to the scope. that mimics a little bit more closely how it will be functioning.

```js
subject.call({ _app: app() }, 'test', {
   // options
});
```